assets/src/edit-story/elements/text/output.js

```diff
@@ -130,7 +118,7 @@ export function TextOutputWithUnits({
 
   const backgroundTextStyle = {
     ...textStyle,
-    color: 'transparent',
+    color: 'transparent !important',
```

Comment:
We definitely can't do `!important` here.

Reply:
Why not?

Reply:
I believe AMP doesn't allow `!important` styles.

Reply:
Oh, that's fair. Then we need to manually find and replace inline color styling though. But it's not too hard.

Reply:
Ping on this.

Reply:
So, I've fixed this by removing inline color styling. It's not a super pretty solution though, so any alternative is very welcome! I tried going the regexp route first, but it ran into too many issues (as HTML is not regular as you know).

Please see c5adfecf

Reply:
Changed it in b63842a4 and a6f390d0 - which is a much better solution.